package.json

```diff
@@ -49,7 +49,6 @@
     "@types/jest-plugin-context": "^2.9.2",
```

Comment:
Does the package-lock.json also need to be pushed and committed as part of the change? As it still has the old dependencies that have been removed.